opennlp-tools/src/main/java/opennlp/tools/ml/AbstractEventTrainer.java

```diff
@@ -92,8 +107,8 @@ public final MaxentModel train(ObjectStream<Event> events) throws IOException {
 
     MaxentModel model = doTrain(indexer);
```

Comment:
I mean the train(ObjectStream<Event> events) could call train(DataIndexer indexer) to do the actual training, otherwise the code is kind of duplicated.